pinot-query-planner/src/test/resources/queries/BasicQueryPlans.json

```diff
@@ -39,7 +39,7 @@
           "Execution Plan",
           "\nLogicalProject(EXPR$0=[dateTrunc('DAY', $4)])",
           "\n  LogicalSort(offset=[0], fetch=[10])",
-          "\n    PinotLogicalSortExchange(distribution=[hash], collation=[[]], isSortOnSender=[false], isSortOnReceiver=[true])",
+          "\n    PinotLogicalSortExchange(distribution=[hash], collation=[[]], isSortOnSender=[false], isSortOnReceiver=[false])",
```

Comment:
This seems correct in my book. we shouldn't run sort unless there's collation key

Reply:
yes that's essentially why i decided to modify the sort exchange for ORDER BY queries to check collation keys to decide whether to sort or not

Reply:
:+1: